### PR TITLE
Initial pull

### PR DESCRIPTION
**Pull request by itsmontoya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Scribe [![GoDoc](https://godoc.org/github.com/hatchify/scribe?status.svg)](https://godoc.org/github.com/hatchify/scribe) ![Status](https://img.shields.io/badge/status-beta-yellow.svg)
+Scribe is an scribe logger helper library a few simple features:
```
Comment (cephalization) [inline README.md]: typo? s/an/a

File: file.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+// NewFile will return a new Scribe
+func NewFile(name, dir string) (fp *File, err error) {
+	var f File
+	f.name = name
+	f.dir = dir
+
+	if err = f.openFile(); err != nil {
+		return
+	}
```
Comment (cephalization) [inline file.go:30]: This pattern with two statements inline with the if is cool, will be stealing this :)

File: file.go [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+	fp = &f
+	return
```
Comment (cephalization) [inline file.go:33]: What does this do? Does go implicitly return `fp` since it is named as a return type?

**Review by cephalization — APPROVED** [review]
I didn't pull down and test but here are a few questions I have.

Take my approval with a grain (grains) of salt :)

File: README.md (hunk 1 above)
Comment (itsmontoya) [reply README.md]: Nice catch!

File: file.go (hunk 3 above)
Comment (itsmontoya) [reply file.go:33]: This is correct! We utilize named return values when possible here. It helps to keep the return values concise. Sometimes the unnamed returns feel ambiguous.

**Review by classic-massok — APPROVED** [review]

File: file.go (hunk 2 above)
Comment (itsmontoya) [reply file.go:30]: For me, it's a really durable way to ensure I don't return my pointer unless everything occurs without error. :)

**Pull request merged**